admin_manual/enterprise_ios_app/publishing_ios_app_9.rst

```diff
@@ -2,4 +2,30 @@
 Publishing Your New Branded iOS App
 ===================================
 
-TODO
\ No newline at end of file
+At last, after following all the previous steps and passing beta testing, your 
+branded iOS app is ready to publish for general distribution on iTunes. You need 
+a Mac computer with Xcode installed (Xcode is a free download).
+
+Download your xcarchive.zip file from your account on 
+`<https://customer.owncloud.com>`_. Your friendly Mac will automatically unpack 
+it and change the name to something like Owncloud iOs Client 29-05-15 
+13.35.xcarchive. Double-click on this file to automatically install it into 
+Xcode. Go to Xcode and you will see it in the Archives listing.
+
+.. figure:: ../images/ios-publish-2.png
+
+Click **Submit to App Store**. If you have not already entered your 
+Developer.apple.com login in Xcode you will be prompted to do it now. Click the 
+Add button.
+
+.. figure:: ../images/ios-publish-3.png
+
+On the next screen enter your account login.
+
+.. figure:: ../images/ios-publish-4.png
+
+After your login is confirmed Xcode opens the Accounts screen. You can close 
+this and go to the next step, where you must choose which developer account to 
+use. Even if you have only one you must select it.
+
```

Comment:
Perhaps, it would be good to add as a pre-requisite that you have to have installed on that computer the distribution certificate as well as the provisioning profile, in this case Provisioning Profile Apple Store. They need 3 of them